lib/xcpretty/formatters/simple.rb

```diff
@@ -8,12 +8,14 @@ class Simple < Formatter
     PASS = "✓"
     FAIL = "✗"
     PENDING = "⧖"
+    MEASURE = "⧗"
```

Comment:
@norio-nomura you can do this one, and you're welcome to convert the others above too